<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonthlyReportGenerator</class>
 <widget class="QDialog" name="MonthlyReportGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>214</width>
    <height>231</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>214</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Report Generator</string>
  </property>
  <widget class="QComboBox" name="monthSelector">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Month :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Year :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="yearSelector">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate </string>
   </property>
  </widget>
  <widget class="QLabel" name="userInfoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
